src/troute-config/troute/config/output_parameters.py

```diff
@@ -28,6 +28,8 @@ class OutputParameters(BaseModel, extra='forbid'):
     # see nwm_routing/output.py :114
     test_output: Optional[FilePath] = None
     stream_output: Optional["StreamOutput"] = None
+    # NOTE: mandatory if writing results to lastobs
+    lastobs_output: Optional[DirectoryPath] = None
```

Comment:
Actually, for output_parameters it is lastobs_output while for compute_parameters it is lastobs_output_folder.  lastobs_output is used when writing lastobs file while lastobs_output_folder is used when reading the file.  I think we can delete lastobs_output_folder not only in compute_parameters.py but also in config yaml file.  Subsequently, we need to change the path to read "lastobs_output_folder" from streamflow_da parameter to output_parameters in output.py.

Reply:
I agree that it can be deleted from compute_parameters.py and yaml file. But I'm pretty sure `lastobs_output_folder` is only used to tell t-route where to write the lastobs file. compute_parameters->streamflow_da->lastobs_file is what is used for reading lastobs. [This](https://github.com/NOAA-OWP/t-route/blob/3fc084a713112d817e65c39abd1eab78c803295f/src/troute-network/troute/DataAssimilation.py#L117-L119) is where reading lastobs happens in this PR